Modify the value of acronym ISS (#1664)
theforeman/foreman-documentation: guides/common/modules/con_how-to-configure-inter-server-sync.adoc

[id="how-to-configure-inter-server-sync_{context}"]
= How to Configure Inter-Satellite Synchronization

> {ProjectName} uses Inter-Satellite Synchronization (ISS) to synchronize content between two {ProjectServer}s in a multi-server {Project} setup.

Reviewer: Can we define it as an attribute so you can write:
```suggestion
{ProjectName} uses {Project}'s {ISS} (ISS) to synchronize content between two {ProjectServer}s including those that are air-gapped.
```

This is mostly because Satellite is downstream branding and an attribute allows us to use `Inter-Server Synchronization` in upstream. It also means you consistently use the same capitalization.

Author: Thanks for this superb idea @ekohl :heart: 

I shall create a new attribute `ISS = Inter-Server Synchronization` to justice upstream and downstream both.

Author: Addressed :+1:
